### PR TITLE
feat: added modal component

### PR DESCRIPTION
**Pull request by Asuraking1n**
feat: added modal component

### CODE REVIEW

File: css/docs.css [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
 .doc-main-body{
     width: 80%;
-    height: 100%;
+    height: auto;
     background-color: transparent;
     overflow-y: auto;
```
Comment (bhavyaj12) [inline css/docs.css:78]: can use shorthand here

File: css/list.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+}
+.list-type-no-bullet{
+    list-style: none;
+}
+.list-type-cap-roman{
+    list-style-type:lower-roman;
+    
+}
+
+
```
Comment (bhavyaj12) [inline css/list.css:97]: remove extra blanks

**Review by bhavyaj12 — COMMENTED** [review]
Looks good, but modal content can be better and inside the modal there is lots of empty space and its content isn't centered

**Comment by Asuraking1n** [comment]
> Looks good, but modal content can be better and inside the modal there is lots of empty space and its content isn't centered

My idea was custom styling, like user can use their designs in modals that's why I left is blank

File: css/list.css (hunk 2 above)
Comment (Asuraking1n) [reply css/list.css:97]: Sure

File: css/docs.css (hunk 1 above)
Comment (Asuraking1n) [reply css/docs.css:78]: By mistake I put 2 height their....I will correct it

**Comment by krgarima** [comment]
The site looks good to me.

**Pull request merged**